README.md

```diff
@@ -26,19 +26,14 @@ Mailhound is a simple server side script for receiving form posts and emailing t
 3. Install mailhound with `npm install -g mailhound`
 4. Add config variable exports to the end of your `~/.profile` file
   1. `export SMTP=smtp://postmaster:pass@smtp.server.com`
-  2. `export KEY:name@domain.com`
+  2. `export ADMIN=name@domain.com`
 5. Reconnect to SSH and run the script `mailhound`
 
 ### Docker Install
 
-1. Create a config file with your environment variables
-
-    ```
-    ADMIN=name@domain.com
-    SMTP=smtp://postmaster:pass@smtp.server.com
-    ```
-
-2. Run docker with `docker run --env-file=<path to the config file> -p 8000:8000 chadfawcett/mailhound`
+1. Create the config file `.env` from `.env.example` with your environment variables
+2. Run docker with `docker run --env-file=<path to the config file .env> -p 8000:8000 chadfawcett/mailhound`
+3. Alternatively run- `docker-compose up`
```

Comment:
Could this be broken up so it's not in the same list? Either add another ###subheading for docker-compose, or just have the list start over at 1.

> 1. ...
> 2. ...
>
> Alternatively, run with docker-compose
> 
> 1. `docker-compose up`